From f47c27ab5c5ee6086ff3a17e434ce677344a1a2c Mon Sep 17 00:00:00 2001
From: Evangelos Foutras <evangelos@foutrelis.com>
Date: Fri, 3 Oct 2014 23:27:02 +0300
Subject: [PATCH] [SF#680]Fix left click on "Move to Workspace" menu

Since the menu popup is initiated by a mouse button release, we need to
specify 0 for the button parameter in the call to gtk_menu_popup().

This fixes left clicking on the "Move to Workspace" sub-menu, which
previously would do nothing.
---
 plugins/launchtaskbar.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/launchtaskbar.c b/plugins/launchtaskbar.c
index fc8e464..d043933 100644
--- a/plugins/launchtaskbar.c
+++ b/plugins/launchtaskbar.c
@@ -2541,7 +2541,7 @@ static gboolean taskbar_task_control_event(GtkWidget * widget, GdkEventButton *
                 GTK_MENU(tb->menu),
                 NULL, NULL,
                 (GtkMenuPositionFunc) taskbar_popup_set_position, (gpointer) visible_task,
-                event->button, event->time);
+                0, event->time);
         }
     }
 
-- 
2.1.2

